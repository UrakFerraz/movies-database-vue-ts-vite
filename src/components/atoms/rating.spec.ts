import { mount } from '@vue/test-utils'
import Rating from './rating.vue'

const props = {
	movie: {vote_average: 8}
}

const wrapper = mount(Rating, { props })

describe('deve ser um componente pontos', () => {
	it('deve renderizar uma div', () => {
		expect(wrapper.html()).toBeDefined
		expect(wrapper.html()).toMatchSnapshot()
		expect(wrapper.html()).toContain('<div class="rating average-rgb--bg"')
		expect(wrapper.html()).toContain('<div class="rate"')
		expect(wrapper.html()).toContain('8</p>')
	})
})
