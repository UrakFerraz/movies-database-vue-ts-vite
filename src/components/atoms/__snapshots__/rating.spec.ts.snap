// Vitest Snapshot v1

exports[`deve ser um componente pontos > deve renderizar uma div 1`] = `
"<div class=\\"rating average-rgb--bg\\" data-v-7dfc6e1a=\\"\\">
  <div class=\\"rate\\" data-v-7dfc6e1a=\\"\\">
    <p data-v-7dfc6e1a=\\"\\">8</p>
  </div>
</div>"
`;
